app/containers/AccountProvider/saga.js

```diff
@@ -165,6 +174,51 @@ export function* isAvailableAction(isValid) {
   yield call(isValid);
 }
 
+const getInvitedTable = async eosService => {
+  const invitedUsers = await eosService.getTableRows(
+    INVITED_USERS_TABLE,
+    INVITED_USERS_SCOPE,
+    undefined,
+    undefined,
+    undefined,
+    undefined,
+    undefined,
+    process.env.EOS_TOKEN_CONTRACT_ACCOUNT,
+  );
+  return invitedUsers;
+};
+
+function* updateRefer(user, eosService) {
+  const invitedUsersInfo = yield call(() => getInvitedTable(eosService));
```

Comment:
Pull details if the user was invited only if cookie does not exit. This is a request that can be avoided in many cases.

Reply:
Added cookie check up

Reply:
you can code simply `yield call(getInvitedTable, eosService)`